README.md

```diff
@@ -16,11 +16,109 @@ var backlight = require('rpi-backlight');
+    res.sendFile('index.html', {root: __dirname })
+});
+
+app.get('/screenOn', (req, res) => {
+    backlight.powerOn();
+    res.send('Screen On' + '<br><a href="/">Back to home page</a>');
+});
+
+app.get('/screenOff', (req, res) => {
+    backlight.powerOff();
+    res.send('Screen OFF' + '<br><a href="/">Back to home page</a>');
+});
+
+// 1 = Screen is OFF
+// 0 = Screen is ON
+app.get('/getPowerValue', (req, res) => {
+    let promise = backlight.getActualPowerValue();
+    promise.then((data) => {
+        res.send('Screen state: ' + data + '<br><a href="/">Back to home page</a>');
+    });    
+});
+
+// The screen get Off at < 10 brightness value
+app.get('/setBrightness/:value', (req, res) => {
+    if (req.params.value > 255) {
+        res.send('ERR: Max value is 255' + '<br><a href="/">Back to home page</a>');
+    } else if (req.params.value < 0) {
+        res.send('ERR: Min value is 0' + '<br><a href="/">Back to home page</a>');
+    } else {
+        backlight.setBrightnessValue(req.params.value);
+        res.send('Screen brightness is now: ' + req.params.value + '<br><a href="/">Back to home page</a>');
+    }
+});
+
+app.get('/goToBrightnessValue/:toValue/:speed', (req, res) => {
+    res.send('Screen brightness: ' + backlight.goToBrightnessValue(req.params.toValue, req.params.speed) + '<br><a href="/">Back to home page</a>'); 
+});
+
+app.get('/setMaxBrightness', (req, res) => {
+    backlight.setMaxBrightness();
+    res.send('Brightness setted to value 255' + '<br><a href="/">Back to home page</a>');
+});
+
+app.get('/getBrightnessValue', (req, res) => {
+    let promise = backlight.getActualBrightness();
+    promise.then((data) => {
+        res.send('Screen brightness: ' + data + '<br><a href="/">Back to home page</a>');
+    });    
+});
+
+/** Server */
+app.listen(3000, () => {
+    console.log("server up on port: 3000 !");
+}); 
+
+```
 
-- [Brightness control](https://github.com/pimterry/rpi-backlight/issues/3)
-- [Power status querying](https://github.com/pimterry/rpi-backlight/issues/1)
-- [Idiot-proofing](https://github.com/pimterry/rpi-backlight/issues/2)
-- ...[your feature here](https://github.com/pimterry/rpi-backlight/issues/new)?
+index.js:
+```html
+<!DOCTYPE html>
+<html >
+    <head>
+    </head>
+    <body>
+        <h1>Let's try this !</h1>
+        <ul>
+            <li><a href="/screenOn">Screen On</a></li>
+            <li><a href="/screenOff">Screen Off</a></li>
+            <li><a href="/getPowerValue">Get screen state</a></li>
+            <li><a href="/setBrightness/50">Set Brigthness value to 50</a></li>
+            <li><a href="/goToBrightnessValue/15/20">Go to Brightness value to 15 with speed 20</a></li>
+            <li><a href="/setMaxBrightness">Set Max Brightness</a></li>
+            <li><a href="/getBrightnessValue">Get actual Brightness</a></li>
+        </ul>
+        <br><br>
+        <h1><a href="" target="__blank">More informations on GitHub!</a></h1>
```

Comment:
Why is there a link here that doesn't go anywhere?

Reply:
Yes, i've forget the link to your depo... https://github.com/pimterry/rpi-backlight/

I thought it might be a good idea to put a link to the source ... especially for issues or other. What do you think?

Reply:
Yep, sounds good to me.

Sorry, one bonus typo: 'informations' should be 'information' here